src/bql.py

```diff
@@ -322,6 +322,15 @@ def execute_phrase(bdb, phrase, bindings=()):
                             'Invalid stattype: %s' % (cmd.stattype))
                     else:
                         stattype = cmd.stattype
+                    # Check that strings are not being modeled as numerical.
+                    if stattype == 'numerical':
+                        cursor = bdb.sql_execute(
+                                'SELECT %s FROM %s' % (qc, qt))
```

Comment:
Response from @riastradh-probcomp 

> @fsaad: The Python and sqlite3 notions of types do not always exactly coincide.
@fsaad: In this case, it is plausible to me that the right heuristic for a variable x is whether selecting unadorned x -- without arithmetic, casts, &c. -- gives a Python unicode object.  That said, I expect it will _never_  give a str object.